hatch.toml

```
@@ -44,6 +44,12 @@ fmt = [
     "black {args:.}",
     "ruff --fix {args:.}",
     "style",
+    "typing",
+    "spelling",
+    "riot",
+    "security",
+    "test",
+    "suitespec-check",
```

What do you think of this, then, which we would invoke with `hatch run lint:checks` ?

```suggestion
]
checks = [
    "style",
    "typing",
    "spelling",
    "riot",
    "security",
    "test",
    "suitespec-check",
```